result/lacplagmember.yaml

```diff
@@ -0,0 +1,200 @@
+openapi: 3.1.0
+
+info:
+  title: LACP results models for LAG member
+  version: ^0.0.0
+
+components:
+  schemas:
+    LacpLagMember.Metrics.Request:
+      description: >-
+        The request to retrieve LACP metrics (statistics) per LAG member.
+      type: object
+      properties:
+        lag_names:
+          description: >-
+            The names of LAG (ports group) for which LACP metrics to be returned.
+            An empty list will return metrics for all LAGs.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - '/components/schemas/Lag/properties/name'
+        lag_member_names:
+          description: >-
+            The names of LAG member (port) for which LACP metrics to be returned.
+            An empty list will return metrics for all LAG members.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - '/components/schemas/Port/properties/name'
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned.
+            Column names 'lag_name' and 'lag_member_name' will always present in returned result set.
+          type: array
+          items:
+            type: string
+            enum:
+              - lag_name
+              - lag_member_name
+              - lacp_in_pkts
+              - lacp_out_pkts
+              - lacp_rx_errors
+
+    LacpLagMember.Metric:
+      description: >-
+        LACP metrics (statistics) per LAG member.
+      type: object
+      properties:
+        lag_name:
+          description: >-
+            The name of a LAG (ports group) configured with LACP.
+          type: string
+        lag_member_name:
+          description: >-
+            The name of a LAG member (port) configured with LACP.
+          type: string
+        lacp_in_pkts:
+          description: >-
+            Number of LACPDUs received.
```

Comment:
A space/hyphen between two words in LACPPDU may easy readable. The same has been occurred other placed too.

Reply:
Attribute description are based on OpenConfig model. It is one word - LACPDU, instead of LACP PDU.
https://github.com/openconfig/public/blob/master/release/models/lacp/openconfig-lacp.yang